### ISSUE TITLE
Looking for new owner

### ISSUE THREAD

**Issue by pi-pi3** [pi-pi3/julia-rs#4]
I won't be maintaining this project anymore and I don't use this account. I'm looking for someone to transfer ownership of the repo and the associated crates ([julia](https://crates.io/crates/julia) and [julia-sys](https://crates.io/crates/julia-sys))

@jaspwr ? Tagging you since you showed recent interest.

**Comment by jaspwr** [comment]
Yes I'm definitely interested. Let me just figure a few things out first (maintaining crates, Julia API, ect.) and I'll let you know if I feel up to it.
